### DISCUSSION TITLE
Enable auto-scrolling in stat_list_card

### DISCUSSION THREAD

**Discussion by psinger** [h2oai/wave#708] [Q&A]
I have a stat list card with many `stat_list_item` so that the box overflows. Is there a way to add a scrollbar if it exceeds the limits?

![image](https://user-images.githubusercontent.com/1677826/112869586-9f412f00-90bd-11eb-9ae9-2df5172d0884.png)

**Comment by lo5** [comment]
You should be seeing a scrollbar if the content overflows the bounds of the card. Do you mind posting your `ui.layout()` definition and a screencap of what the card looks like in the context of the page?

**Reply by psinger** [reply]
Zones are defined as this:

zones=[
   ```
 ui.zone('title'),
    ui.zone('top', size="410px", direction=ui.ZoneDirection.ROW, zones=[
        ui.zone('top_left', size='50%', direction=ui.ZoneDirection.ROW),
        ui.zone('top_right', size='50%', direction=ui.ZoneDirection.ROW),
    ]),
    ui.zone('middle', size="205px", direction=ui.ZoneDirection.ROW, zones=[
        ui.zone('middle_left', size='50%', direction=ui.ZoneDirection.ROW),
        ui.zone('middle_right', size='50%', direction=ui.ZoneDirection.ROW),
    ]),
    ui.zone('bottom', size="205px", direction=ui.ZoneDirection.ROW, zones=[
        ui.zone('bottom_left', size='50%', direction=ui.ZoneDirection.ROW),
        ui.zone('bottom_right', size='50%', direction=ui.ZoneDirection.ROW),
    ]),
]
```

Card:

```
for col in cols:
    items.append(ui.stat_list_item(label=col, value=str(val)))
q.page["experiment_bottomright_0"] = ui.stat_list_card(
    box="bottom_right",
    title="xxx",
    items=items
)
```

Context:

![image](https://user-images.githubusercontent.com/1677826/112901547-32418f80-90e5-11eb-943b-d3e2159bf463.png)
